roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -150,12 +150,12 @@
     mode: 0750
   with_items: "{{ kafka_broker_final_properties['log.dirs'].split(',') }}"
 
-- name: Set Permissions on Data Dir files
-  file:
-    path: "{{item}}"
-    owner: "{{kafka_broker_user}}"
-    group: "{{kafka_broker_group}}"
-    recurse: true
+- name: Set Permissions on Data Dir files  # noqa deprecated-command-syntax
+  # Have to use command + chown instead of the Ansible file module here as
+  # it seems like the file module can't handle files modified/deleted while running.
+  # See https://github.com/confluentinc/cp-ansible/pull/903 for details.
```

Comment:
```suggestion
  # Have to use command + chown instead of the Ansible file module here as
  # it seems like the file module can't handle files modified/deleted while running. 
  # This task can fail on a very active cluster. See https://github.com/confluentinc/cp-ansible/pull/903 for details.
```